README.md

```diff
@@ -71,6 +71,7 @@ The available options are:
   appearing in parser input and raw bytes in source code, and how Lua escape
   sequences in JavaScript strings should be interpreted. See the
   [Encoding modes](#encoding-modes) section below for more information.
+- `preserveParens: false` If this option is true, parenthesized expressions are represented by (non-standard) ParenthesizedExpression nodes that have a single expression property containing the expression inside parentheses.
```

Comment:
Qualifying them as ‘non-standard’ is meaningless, as none of those tree node names are actually standardised anywhere. (They were ‘inspired’ by esprima, which took them straight from the grammar productions of the ECMA-262 specification. Since Lua is not ECMAScript, this is pretty arbitrary, but they had to be named somehow anyway, and those names as good as any other.)

Reply:
My bad, i just copied the description acorn gave it.

Reply:
Fixed. Hope this is a better wording, if not please suggest.